Admin/AbstractAdmin.php

```diff
@@ -2382,6 +2398,13 @@ public function setBaseCodeRoute($baseCodeRoute)
      */
     public function getBaseCodeRoute()
     {
+        if ($this->isChild()) {
+            return $this->getParent()->getBaseCodeRoute().'|'.$this->getCode();
+        }
```

Comment:
That's it. Otherwise, since child admins are added (via `addChild(...)`) only once, they always have one level of nesting (i.e. one `|`) on their base code route.